Trace/Hardware/Device/DeviceModel.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  DeviceModel.swift
+//  Trace
+//
+//  Created by Mukund Agarwal on 23/07/2020.
+//  Copyright © 2020 Bitrise. All rights reserved.
+//
+
+import UIKit
+
+public extension UIDevice {
+    
+    /// pares the device name as the standard name
+    var modelName: String {
```

Comment:
Haven't added test for this as that could change based on the target its being tested on.

Reply:
worth having a basic one that checks if the value is NIL. that way if anything was to happen in the future it would break/crash the unit test runs.

Reply:
Added. Thanks.